src/smart-components/group/group.js

```diff
@@ -86,10 +87,13 @@ const Group = ({
     <DropdownItem
       component={
         <Link
-          to={(location.pathname.includes('members') ? routes['group-detail-members-remove'] : routes['group-detail-roles-remove']).replace(
-            ':uuid',
-            uuid
-          )}
+          to={() => {
+            onDelete();
```

Comment:
@PanSpagetka did you try hooking the onDelete callback to `onClick` event? That might be the "correct" way of doing it. If it works.

Reply:
Yep, that's better. For some reason I thought it doesn't work, but actually it does.